components/ipfs/test/BUILD.gn

```diff
@@ -50,6 +50,8 @@ source_set("brave_ipfs_browser_tests") {
     sources = [
       "//brave/components/ipfs/brave_ipfs_client_updater_browsertest.cc",
       "//brave/components/ipfs/ipfs_navigation_throttle_browsertest.cc",
+      "//brave/components/ipfs/ipfs_not_connected_page_browsertest.cc",
+      "//brave/components/ipfs/ipfs_onboarding_page_browsertest.cc",
```

Comment:
Now that we're adding more browser tests, probably better fix the TODO of moving these browser tests which have layering violations into the proper location at some point.

Reply:
done